CMakeLists.txt

```diff
@@ -544,30 +544,37 @@ endif ( GLib2_VERSION AND GLib2_VERSION VERSION_LESS "2.26.0" )
 unset ( LIBSNDFILE_SUPPORT CACHE )
 unset ( LIBSNDFILE_HASVORBIS CACHE )
 if ( enable-libsndfile )
-    find_package ( SndFile ${LIBSNDFILE_MINIMUM_VERSION} )
+    #set(CMAKE_FIND_DEBUG_MODE ON)
+    find_package ( SndFile ${LIBSNDFILE_MINIMUM_VERSION} QUIET )
     set ( LIBSNDFILE_SUPPORT ${SndFile_FOUND} )
     if ( LIBSNDFILE_SUPPORT )
+        message ( STATUS "Found libSndFile: ${SndFile_VERSION}" )
+        #cmake_print_properties(TARGETS SndFile::sndfile PROPERTIES LOCATION INTERFACE_INCLUDE_DIRECTORIES IMPORTED_CONFIGURATIONS)
+        cmake_print_variables(SndFile_WITH_EXTERNAL_LIBS SndFile_WITH_MPEG)
         list( APPEND PC_REQUIRES_PRIV "sndfile")
         if ( SndFile_WITH_EXTERNAL_LIBS )
             set ( LIBSNDFILE_HASVORBIS 1 )
         else (SndFile_WITH_EXTERNAL_LIBS)
             message ( NOTICE "Seems like libsndfile was compiled without OGG/Vorbis support." )
         endif (SndFile_WITH_EXTERNAL_LIBS)
+	else ( LIBSNDFILE_SUPPORT )
```

Comment:
Could you pls. remove the tabulator here to fix indentation?

Reply:
fixed